sqlmesh/core/console.py

```diff
@@ -613,22 +614,35 @@ def show_model_difference_summary(
             no_diff: Hide the actual SQL differences.
         """
         if context_diff.is_new_environment:
-            self._print(
-                Tree(
-                    f"[bold]New environment `{context_diff.environment}` will be created from `{context_diff.create_from}`"
-                )
+            msg = (
+                f"Initializing `{c.PROD}` environment"
```

Comment:
I think this message is misleading. We're just showing the plan summary here and so nothing gets "initialized" or "created" at this point. This is why it used to say "will be".

Reply:
so then you suggest? "`prod` environment will be created"